index.js

```diff
@@ -1 +1,19 @@
 // Your code here
+
+let getTime = function getTime() {
+  let date = new Date;
+  let hours = date.getHours();
+  let minutes = date.getMinutes();
+  if (minutes < 10) {
+    minutes = '0' + minutes;
+  }
+
+  let seconds =date.getSeconds();
+  if (seconds < 10) {
+    seconds = '0' + seconds;
+  }
+
+$('#clock').html(`The time is...<h1> ${hours}:${minutes}:${seconds} </h1>`);
+};
+
```

Comment:
A couple of things:

1.  You should put the `setInterval` call inside of `$(document).ready(() => { ... });`
2.  You have jQuery loading **after** your `index.js` file.  

It works because by the time `setInterval` fires jQuery and the HTML are loaded, but wouldn't be guaranteed in a deployed app.